src/repl.rs

```diff
@@ -71,9 +71,6 @@ fn prepare_home_dir() -> PathBuf {
 fn main() {
     let cli_opt: WalletOpt = WalletOpt::from_args();
 
-    let level = LevelFilter::from_str(cli_opt.log_level.as_str()).unwrap_or(LevelFilter::Info);
-    env_logger::builder().filter_level(level).init();
```

Comment:
I think we need default initialization of the logger?

`env_logger::init();`

Reply:
Oh ya! I should have caught that. Fixed now.